bot/build.sh

```diff
@@ -21,12 +21,14 @@
 # stop as soon as something fails
 set -e
 
+# Make sure we are referring to software-layer as working directory
+topdir=$(dirname $(dirname $(realpath $0)))
```

Comment:
We should probably give this a more descriptive name, to avoid mistakes, like:

```suggestion
software_layer_dir=$(dirname $(dirname $(realpath $0)))

Reply:
Addressed in 211840f